sycl/include/sycl/info/device_traits.def

```diff
@@ -6,233 +6,236 @@
-                         PI_DEVICE_INFO_GPU_EU_SIMD_WIDTH)
-__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_slices, pi_uint32,
-                         PI_DEVICE_INFO_GPU_SLICES)
-__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_subslices_per_slice, pi_uint32,
-                         PI_DEVICE_INFO_GPU_SUBSLICES_PER_SLICE)
-__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_eu_count_per_subslice, pi_uint32,
-                         PI_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE)
-__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_hw_threads_per_eu, pi_uint32,
-                         PI_DEVICE_INFO_GPU_HW_THREADS_PER_EU)
+                         UR_DEVICE_INFO_PCI_ADDRESS)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_eu_count, uint32_t,
+                         UR_DEVICE_INFO_GPU_EU_COUNT)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_eu_simd_width, uint32_t,
+                         UR_DEVICE_INFO_GPU_EU_SIMD_WIDTH)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_slices, uint32_t,
+                         UR_DEVICE_INFO_GPU_EU_SLICES)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_subslices_per_slice, uint32_t,
+                         UR_DEVICE_INFO_GPU_SUBSLICES_PER_SLICE)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_eu_count_per_subslice, uint32_t,
+                         UR_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_gpu_hw_threads_per_eu, uint32_t,
+                         UR_DEVICE_INFO_GPU_HW_THREADS_PER_EU)
 __SYCL_PARAM_TRAITS_SPEC(device, ext_intel_device_info_uuid, detail::uuid_type,
-                         PI_DEVICE_INFO_UUID)
-__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_max_mem_bandwidth, pi_uint64,
-                         PI_DEVICE_INFO_MAX_MEM_BANDWIDTH)
-__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_max_work_groups_1d, id<1>,
-                         PI_EXT_ONEAPI_DEVICE_INFO_MAX_WORK_GROUPS_1D)
-__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_max_work_groups_2d, id<2>,
-                         PI_EXT_ONEAPI_DEVICE_INFO_MAX_WORK_GROUPS_2D)
+                         UR_DEVICE_INFO_UUID)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_max_mem_bandwidth, uint64_t,
+                         UR_DEVICE_INFO_MAX_MEMORY_BANDWIDTH)
+
+__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_max_work_groups_1d, id<1>, __SYCL_TRAIT_HANDLED_IN_RT)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_max_work_groups_2d, id<2>, __SYCL_TRAIT_HANDLED_IN_RT)
 __SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_max_work_groups_3d, id<3>,
-                         PI_EXT_ONEAPI_DEVICE_INFO_MAX_WORK_GROUPS_3D)
-__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_max_global_work_groups, size_t,
-                         PI_EXT_ONEAPI_DEVICE_INFO_MAX_GLOBAL_WORK_GROUPS)
+                         UR_DEVICE_INFO_MAX_WORK_GROUPS_3D)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_max_global_work_groups, size_t, __SYCL_TRAIT_HANDLED_IN_RT)
 __SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_cuda_cluster_group, bool,
```

Comment:
I wonder if we should check uses of `bool` within info queries. `bool` is problematic because of its unspecified size and I've seen some `ur_bool_t` typedef so I wonder if that is what should be used once we move to UR instead of just `bool`?

Reply:
Yeah, that should be `ur_bool_t` fixed that

Reply:
This turned to be problematic atm, and will need more investigation. so, I made a follow up task in this [issue](https://github.com/intel/llvm/issues/14598) for that